Added Exception Handling entry to `document/core/appendix/changes.rst` (#267)
WebAssembly/exception-handling: document/core/appendix/changes.rst

* New injection/projection :ref:`vector instructions <syntax-instr-vec>`: :math:`\K{i}\!N\!\K{x}\!M\!\K{.splat}`, :math:`\K{f}\!N\!\K{x}\!M\!\K{.splat}`, :math:`\K{i}\!N\!\K{x}\!M\!\K{.bitmask}`

.. index:: instructions, exception, tag type, tag, handler

Exception Handling
..................

> Added tag type, instructions that throw exceptions of a tag type, and instructions that handle exceptions. [#proposal-eh]_

Reviewer: ```suggestion
Added tag definitions, imports, and exports, and instructions to throw and catch exceptions [#proposal-eh]_
```

> * New :ref:`tag type <syntax-tagtype>`: :math:`[t^\ast]\to[]`-

Reviewer: I'd replace this with tag definitions, imports, and exports. Tag types are more of an auxiliary notion to support these constructs.

> * New :ref:`tag section <binary-tagsec>` in binary format.

Reviewer: Perhaps move this to the end of the list.

* New exception throwing :ref:`control instructions <syntax-instr-control>`: :math:`\THROW` and :math:`\RETHROW`.

> * New handler :ref:`control instructions <syntax-instr-control>`: :math:`\TRY~\X{bt}~\instr_1^\ast~(\CATCH~x~\instr_2^\ast)^\ast~(\CATCHALL~\instr_3^\ast)^?\END`, :math:`\TRY~\X{bt}~\instr^\ast~\DELEGATE~l`.

Reviewer: ```suggestion
* New handler :ref:`control instructions <syntax-instr-control>`: :math:`(\TRY~\X{bt}~\instr_1^\ast~(\CATCH~x~\instr_2^\ast)^\ast~(\CATCHALL~\instr_3^\ast)^?\END)` and :math:`(\TRY~\X{bt}~\instr^\ast~\DELEGATE~l)`.
```

> * New uncaught exception :ref:`result <syntax-result>`.

Reviewer: I'd remove this, it's more like an implication, not a feature.

.. [#proposal-vectype]
   https://github.com/WebAssembly/spec/tree/main/proposals/simd/

> .. [#proposal-eh]

Reviewer: ```suggestion
.. [#proposal-exceptions]
```
